tests/functional/incremental-fetching-date-with-state/expected/data/out/tables/in.c-main.auto-increment-timestamp.csv.manifest

```diff
@@ -100,11 +96,11 @@
             },
             {
                 "key": "KBC.uniqueKey",
-                "value": true
+                "value": false
```

Comment:
Toto odbc nereportuje, ale neviem o tom, ze by sa to niekde v platforme pouzivalo.